File: src/main/java/com/goudourasv/data/users/UsersMapper.java
package com.goudourasv.data.users;

import com.goudourasv.domain.users.User;

public class UsersMapper {

    public static User toUser(UserEntity userEntity) {
        User user = new User(userEntity.getId(), userEntity.getFirstName(), userEntity.getLastName(), userEntity.getEmail());
        return user;
    }
}

File: src/main/java/com/goudourasv/data/users/UsersMapper.kt
package com.goudourasv.data.users

import com.goudourasv.domain.users.User
import com.goudourasv.http.users.dto.UserCreate

fun UserEntity.toUser(): User {
    return User(
        id = this.id!!,
        firstName = this.firstName,
        lastName = this.lastName,
        email = this.email,
    )
}
    fun UserCreate.toUserEntity(): UserEntity{
        return UserEntity(
            firstName = this.firstName,
            lastName = this.lastName,
            email = this.email,
        )
    }
